src/pages/api/repair-request.ts

```diff
@@ -33,15 +33,11 @@ const createRepairRequest = async (
   }
 
   try {
-    const { eventId, description, itemType, itemBrand, images } = response.data;
-    const repairRequest = await repairRequestModel.insert(
-      eventId,
-      description,
-      itemType,
-      itemBrand,
-      "mock_user", // TODO: change this once we get userID
-      images
-    );
+    const repairRequest = await repairRequestService.insert({
+      ...response.data,
```

Comment:
I think the variable name `response` is a little confusing since there's also `res`. Could it be renamed to something like `prismaRequestBody` or somethin